golang/.gitignore

```diff
@@ -3,6 +3,4 @@
 .DS_Store
 build
 Godeps/_workspace
-messagetype_string.go
-reqresreaderstate_string.go
-reqreswriterstate_string.go
+*_string.go
```

Comment:
I was under the impression that you're supposed to check the generated code into the repository?

At least, that's what http://blog.golang.org/generate says.